x-pack/plugins/ml/public/application/jobs/new_job/job_from_lens/utils.ts

```diff
@@ -35,6 +36,8 @@ export const COMPATIBLE_LAYER_TYPE: XYDataLayerConfig['layerType'] = layerTypes.
 
 export const COMPATIBLE_VISUALIZATION = 'lnsXY';
 
+export const COMPATIBLE_SPLIT_FIELD_TYPES: DataType[] = ['string', 'ip'];
```

Comment:
could you please import `KBN_FIELD_TYPES` from `@kbn/data-plugin/public`?

```suggestion
export const COMPATIBLE_SPLIT_FIELD_TYPES: DataType[] = [KBN_FIELD_TYPES.STRING, KBN_FIELD_TYPES.IP];
```

Reply:
Updated here https://github.com/elastic/kibana/pull/132753/commits/afa5ac450cce3a7ebbbc00f2dbbae8f06af1bb82
Also switched to using our job aggregation constants.